hooks/use-manage-video.tsx

```
@@ -0,0 +1,70 @@
+  const [isDownloading, setIsDownloading] = useState(false);
+
+  const downloadVideo = async () => {
+    setIsDownloading(true);
+    if (!video) return toast.error("No video is set");
+
+    try {
+      toast.promise(getDownloadDataById(video.cloudflareId), {
+        loading: "Downloading video...",
+        success: (data) => {
+          const link = document.createElement("a");
+          if (!(data as any)?.default?.url)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)